device/.cargo/config.toml

```diff
@@ -3,7 +3,7 @@ target = "riscv32imc-unknown-none-elf"
 rustflags = ["-C", "link-arg=-Tlinkall.x"]
 
 [target.riscv32imc-unknown-none-elf]
-runner = "espflash flash --baud 921600 --partition-table partitions.csv"
+runner = "espflash flash --baud 921600 --partition-table partitions.csv --bootloader bootloader-dev.bin"
```

Comment:
should we also add this to `just flash`?
Remind me, this `bootloader-dev` doesnt have secure boot keys right?